### PR TITLE
chore: converted to md in developer/16.0/guides/develop/touch-keyboard…

### PR DESCRIPTION
**Pull request by Meng-Heng**
There are two files in the folder that were PHP files so I went ahead and converted it to markdown. If there is anything I need to change, I will be happy to do so.

For reference - https://help.keyman.com/developer/16.0/guides/develop/touch-keyboard-tutorial/

This section is now changed to a table
> |             |          |     |          |     |     |          |     |          |
> |-------------|----------|-----|----------|-----|-----|----------|-----|----------|
> | `        ɓ` | `U+0253` | `Ɓ` | `U+0181` |     | `ɗ` | `U+0257` | `Ɗ` | `U+018A` |
> | `        ŋ` | `U+014B` | `Ŋ` | `U+014A` |     | `ƴ` | `U+01B4` | `Ƴ` | `U+01B3` |

I cannot find anything that can reimplement the box `<kbd>` so this is the closest thing I found

> ```
> ago) used the ```/``` key as the deadkey.
> ***`/`*** ***`b`*** gave ɓ,
> ***`/`*** ***`d`*** gave ɗ, and so on.
> ```

### CODE REVIEW

**Comment by mcdurdin** [comment]
Looks good so far!

> This section is now changed to a table

:+1:

> I cannot find anything that can reimplement the box `<kbd>` so this is the closest thing I found

You can actually continue to use `<kbd>x</kbd>` in the Markdown content. So do go ahead and update to use that tag :grin:

File: developer/16.0/guides/develop/touch-keyboard-tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+Fulfulde is usually written using the Latin alphabet, but there are a
+few characters added to the usual list of alphabetical characters. So
+this is a useful example for any language wanting to add a few
+characters to the standard Latin keyboard. Here are the characters we
+are going to add, together with their Unicode values.
+
+|             |          |     |          |     |     |          |     |          |
+|-------------|----------|-----|----------|-----|-----|----------|-----|----------|
+| `        ɓ` | `U+0253` | `Ɓ` | `U+0181` |     | `ɗ` | `U+0257` | `Ɗ` | `U+018A` |
+| `        ŋ` | `U+014B` | `Ŋ` | `U+014A` |     | `ƴ` | `U+01B4` | `Ƴ` | `U+01B3` |
```
Comment (mcdurdin) [inline developer/16.0/guides/develop/touch-keyboard-tutorial/index.md:31]:
```suggestion
|     |          |     |          |     |     |          |     |          |
|-----|----------|-----|----------|-----|-----|----------|-----|----------|
| `ɓ` | `U+0253` | `Ɓ` | `U+0181` |     | `ɗ` | `U+0257` | `Ɗ` | `U+018A` |
| `ŋ` | `U+014B` | `Ŋ` | `U+014A` |     | `ƴ` | `U+01B4` | `Ƴ` | `U+01B3` |
```

File: developer/16.0/guides/develop/touch-keyboard-tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+Here we will likely want to proceed differently than with the desktop
+keyboard. The usual approach for a desktop keyboard is to create a
+deadkey. The keyboards I have used for Fulfulde (going back to decades
+ago) used the ```/``` key as the deadkey.
```
Comment (mcdurdin) [inline developer/16.0/guides/develop/touch-keyboard-tutorial/index.md:36]:
```suggestion
ago) used the <kbd>/</kbd> key as the deadkey.
```

And others in the same way. `<kbd>` is a valid tag to use in Markdown

**Review by mcdurdin — CHANGES_REQUESTED** [review]

**Comment by Meng-Heng** [comment]
> You can actually continue to use <kbd>x</kbd> in the Markdown content. So do go ahead and update to use that tag 😁

Now, it has changed. I was going to keep <kbd> but thinking it was a tag so I removed it 😅

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**